crates/cast/tests/e2e/show_config.rs

```diff
@@ -0,0 +1,77 @@
+use crate::helpers::runner::runner;
+use indoc::indoc;
+
+#[tokio::test]
+async fn test_show_config_from_scarb_toml() {
+    let args = vec![
+        "--path-to-scarb-toml",
+        "tests/data/show_config/all_Scarb.toml",
+        "--profile",
+        "profile1",
+        "show-config",
+    ];
+
+    let snapbox = runner(&args);
+
+    snapbox.assert().success().stdout_eq(indoc! {r#"
+        command: show-config
+        account: user1
```

Comment:
when there is `keystore`, the account is a path to account file (this is something different than accounts file)

show-config should:

- not show accounts_file_path when keystore is non empty, the account then should be a path to account file
- show accouns_file_path when keystore is empty, the account then is an account name from accounts file

let's adjust the outputs to this :+1:

Reply:
> show accouns_file_path when keystore is empty, the account then is an account name from accounts file

do i need to do validation that `account_name` is present inside `accounts_file`?

Reply:
no, this is already done elsewhere :)